builder/Makefile

```diff
@@ -73,7 +73,7 @@ $(builder.image.out): $(PACK) $(builder.config)
 
 builder.image: $(builder.image.out) $(builder.image.out).sha256
 builder.image.local: builder.flags=--pull-policy if-not-present
-builder.image.local: builder.image
+builder.image.local: builder.clean builder.image
```

Comment:
Hmm I get the convenience of putting clean here, but perhaps we can solve this by requiring the buildpacks to be built instead?

Reply:
The reasoning behind this is less about the big picture and more about just the task of building a local builder. For instance, it made sense that _anytime_ a dev wants to build a local builder, they would want to make sure any artifacts from the previous builder are cleared out, to guarantee a fresh local build. Not the case for non-local builders. Thoughts?

Reply:
I couldn't decide where to put the answer as it was related to this and another comment. https://github.com/vmware-tanzu/function-buildpacks-for-knative/pull/140#discussion_r989470056